Client/Frontend/Browser/NewTabAccessors.swift

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Nit: If there is a chance that we would be using these `Accessors` outside of Browser, then I think I would prefer to have an 'Accessors' folder where all accessors could be found. Kinda like Extensions, or Widgets or Utils. Might make navigation easier.